openwisp_monitoring/device/admin.py

```diff
@@ -160,56 +167,68 @@ def get_form(self, request, obj=None, **kwargs):
         return super().get_form(request, obj, **kwargs)
 
     def get_fields(self, request, obj=None):
-        fields = super().get_fields(request, obj)
+        fields = list(super().get_fields(request, obj))
         if not obj or obj.monitoring.status in ['ok', 'unknown']:
             return fields
-        fields = list(fields)
+        fields.insert(fields.index('last_ip'), 'health_status')
```

Comment:
this won't be executed if the if above is triggered, which was not how it worked before (always added), please rever this